modules/administration-guide/partials/con_che-devfile-registry.adoc

```diff
@@ -1,22 +1,15 @@
 // Module included in the following assemblies:
 //
-// {prod-id-short}-workspace-controller
+// {prod-id-short}-workspace-controller-with-che-server
 
 [id="{prod-id-short}-devfile-registry_{context}"]
 = {prod-short} Devfile registry
 
-The {prod-short} devfile registry is a service that provides a list of {prod-short} stacks to create ready-to-use workspaces. This list of stacks is used in the  *Dashboard* -> *Create Workspace* window. The devfile registry runs in a container and can be deployed wherever the user dashboard can connect.
+The {prod-short} devfile registry is a service that provides a list of {prod-short} samples to create ready-to-use workspaces. This list of samples is used in the *Dashboard* -> *Create Workspace* window. The devfile registry runs in a container and can be deployed wherever the user dashboard can connect.
```

Comment:
Can I only use the samples if I'm in the *Create Workspace* window? Or can I access the samples from the *Create Workspace* window and use them else where? 

If the first, then leave phrasing as is. Otherwise, consider rephrasing to something like, ```The list of samples is located in the *Dashboard -> Create Workspace* window```.